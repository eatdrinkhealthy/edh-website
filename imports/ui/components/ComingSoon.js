import React from "react";
import { comingSoonStyle } from "../styles";

const ComingSoon = () => (
  <div className={comingSoonStyle.className}>Coming Soon</div>
);

ComingSoon.style = comingSoonStyle;

export default ComingSoon;
