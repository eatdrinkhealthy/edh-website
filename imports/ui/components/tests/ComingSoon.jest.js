/* eslint-env jest */
/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */

import React from "react";
import renderer from "react-test-renderer";
import {
  shallow,
} from "enzyme";
import ComingSoon from "../ComingSoon";

describe("<ComingSoon />", function () {
  it("render matches snapshot", function () {
    const tree = renderer.create(<ComingSoon />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("css matches snapshot", function () {
    // informative when doing TDD (provides useful messaging before snapshot is finalized)
    expect(shallow(<ComingSoon />).hasClass(ComingSoon.style.className)).toBe(true);

    // checking snapshot
    expect(JSON.stringify(ComingSoon.style, null, 2)).toMatchSnapshot();
  });
});
