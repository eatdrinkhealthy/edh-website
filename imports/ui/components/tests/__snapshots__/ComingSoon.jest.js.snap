// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ComingSoon /> matches render snapshot 1`] = `
<div
  className="css-cdesk3"
>
  Coming Soon
</div>
`;

exports[`<ComingSoon /> matches style / css snapshot 1`] = `
"[
  {
    \\"rules\\": {
      \\"fontFamily\\": \\"Montserrat\\",
      \\"paddingTop\\": \\"30px\\",
      \\"fontSize\\": \\"20px\\",
      \\"fontWeight\\": \\"bold\\",
      \\"textAlign\\": \\"center\\",
      \\"color\\": \\"white\\",
      \\"paddingBottom\\": \\"30px\\",
      \\"borderBottom\\": \\"1px solid rgba(255,255,255,0.6)\\",
      \\"display\\": \\"block\\",
      \\"width\\": \\"226px\\",
      \\"margin\\": \\"0 auto\\"
    },
    \\"className\\": \\"css-cdesk3\\"
  }
]"
`;
